src/main/java/net/sf/jabref/model/database/DuplicationChecker.java

```diff
@@ -2,62 +2,43 @@
 
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Optional;
 
-import org.apache.commons.logging.Log;
-import org.apache.commons.logging.LogFactory;
+import net.sf.jabref.model.database.event.EntryAddedEvent;
+import net.sf.jabref.model.database.event.EntryRemovedEvent;
+import net.sf.jabref.model.entry.BibEntry;
+import net.sf.jabref.model.entry.event.FieldChangedEvent;
+
+import com.google.common.eventbus.Subscribe;
 
 /**
  * Determines which bibtex cite keys are duplicates in a single {@link BibDatabase}.
  */
-class DuplicationChecker {
-
-    private static final Log LOGGER = LogFactory.getLog(DuplicationChecker.class);
+public class DuplicationChecker {
 
-    // use a map instead of a set since i need to know how many of each key is in there
+    /** use a map instead of a set since I need to know how many of each key is in there */
     private final Map<String, Integer> allKeys = new HashMap<>();
 
+
```

Comment:
I know that I got many comments for empty lines like this.
But I think in this case it's okay to leave it.